### PR TITLE
MOTD rework

### PR DESCRIPTION
**Pull request by Heartfyre**
motd2

Reworked window, added fancy scrollbar, removed MOTDStyles

### CODE REVIEW

File: game/hud/src/components/MOTD.tsx [hunk 1]
```diff
@@ -111,27 +198,36 @@ class MOTD extends React.Component<MOTDProps, MOTDState> {
...
-              <Footer>
-                <DismissButton onClick={this.hideDelay}>Dismiss For 24h</DismissButton>
-              </Footer>
+                    : 'Welcome to Camelot Unchained'
+                  }</h4>
+                </MOTDMotdTitle>
+                <MOTDContent>
+                      {
+                        gqlData && gqlData.motd && gqlData.motd[0]
+                        ? <div key='100' dangerouslySetInnerHTML={{ __html: gqlData.motd[0].htmlContent }} />
```
Comment (Ajackster) [inline game/hud/src/components/MOTD.tsx]: What is this key for? I think you can just get rid of it.

File: game/hud/src/components/MOTD.tsx [hunk 2]
```diff
@@ -145,8 +241,8 @@ class MOTD extends React.Component<MOTDProps, MOTDState> {
   private hideDelay = (): void => {
     this.hide();
     const hideDelayStart: Date = new Date();
-    localStorage.setItem('cse-MOTD-hide-start', JSON.stringify(hideDelayStart));
+    localStorage.setItem('cse-welcome-hide-start', JSON.stringify(hideDelayStart));
```
Comment (Ajackster) [inline game/hud/src/components/MOTD.tsx]: Leave this as cse-MOTD-hide-start

File: game/hud/src/components/MOTD.tsx [hunk 3]
```diff
@@ -145,8 +241,8 @@ class MOTD extends React.Component<MOTDProps, MOTDState> {
...
   }
 }
 
-export default MOTD;
+export default Welcome;
```
Comment (Ajackster) [inline game/hud/src/components/MOTD.tsx]: Leave the component name as MOTD

**Comment by Heartfyre** [comment]
Added the requested changes. PR got updated.

**Comment by Ajackster** [comment]
Thanks @Heartfyre , looks great! This has been merged [here](https://github.com/CUModSquad/Camelot-Unchained/commit/927f3fc55e902e65670dc952df6968b1786adae8)

**Pull request closed without merging**